test/fixtures/wpt/performance-timeline/po-observe-type.any.js

```diff
@@ -33,6 +33,13 @@ test(() => {
   });
 }, "Calling observe() with type and entryTypes should throw a TypeError");
 
+test(() => {
```

Comment:
Changes in WPT should be submitted to the upstream first: https://github.com/web-platform-tests/wpt.

In the meantime, you can move this test case to `test/parallel/` directory. That's where Node.js' test cases are located at.

Reply:
Have the changes been upstreamed?